packages/components/src/components/accordion/shadow.tsx

```diff
@@ -27,14 +36,27 @@ featureHint(`[KolAccordion] Tab-Sperre des Inhalts im geschlossenen Zustand.`);
 	},
 	shadow: true,
 })
-export class KolAccordion implements AccordionAPI {
-	@Element() private readonly host?: HTMLKolDetailsElement;
+export class KolAccordion implements AccordionAPI, FocusableElement {
 	private readonly nonce = nonce();
+	private buttonWcRef?: HTMLKolButtonWcElement;
 
 	private readonly catchRef = (ref?: HTMLKolButtonWcElement) => {
-		propagateFocus(this.host, ref);
+		this.buttonWcRef = ref;
 	};
 
+	/**
+	 * @deprecated Use kolFocus instead.
+	 */
+	@Method()
```

Comment:
Hier bin ich mir unsicher, ob das die beste Methode ist und wäre für Input bzw. weitere Ideen und Meinungen dankbar.

Zuvor war `.focus` praktisch als versteckte Methode, als einfache Referenz vom nativen Element auf das Root-Element der Webcomponent definiert. Das wurde vom Stencil-Compiler nicht weiter erkannt oder berücksichtigt.

Ich habe `.focus` jetzt bewusst als richtige `@Method` definiert, weil sie dadurch korrekt (mit dem `deprecated` Hinweis) in der Readme gelistet wird und auch in den Types (relevant für IDEs, TS-Compiler) als deprecated markiert wird.

Eigentlich halte ich diese Lösung für korrekt, es führt aber dazu, dass unsere Konsole mit Warnings zugespammt wird:
<img width="848" alt="image" src="https://github.com/public-ui/kolibri/assets/971072/10a94f67-6006-4379-aba7-88e51c83ddf7">

Reply:
Wir haben jetzt zwei Optionen:

1. `focus` überschreiben und die Warnung hinnehmen
2. `focus` in Ruhe lassen und gegebenenfalls Fehler in den Anwendungen in Kauf nehmen, da `focus` statt `kolFocus` verwendet wurde.

Ich würde dann lieber die Warnung hinnehmen.

Reply:
Da hiervon nicht die Nachnutzenden betroffen sind, würde ich Option 1 empfehlen.